Examples/Sources/MacroExamples/Interface/ExtensionMacros.swift

```diff
@@ -14,3 +14,11 @@
 
 @attached(extension, conformances: Equatable)
 public macro equatable() = #externalMacro(module: "MacroExamplesImplementation", type: "EquatableExtensionMacro")
+
+// MARK: - Protocol Default Impl
+
+@attached(extension, names: arbitrary)
+public macro defaultImpl() = #externalMacro(
```

Comment:
I would like to avoid `Impl` here since it’s an abbreviation. What about `defaultFatalErrorImplementation`?

Reply:
I always struggle with names; thank you for the suggestion. I've renamed macro to `DefaultFatalErrorImplementationMacro` and changed the attribute name to `defaultFatalErrorImplementation`.